app/src/main/java/com/framgia/englishconversation/screen/timeline/TimelineViewModel.java

```diff
@@ -6,9 +6,11 @@
 import android.databinding.ObservableField;
 import android.os.Bundle;
 import com.android.databinding.library.baseAdapters.BR;
+import com.framgia.englishconversation.data.model.MediaModel;
 import com.framgia.englishconversation.data.model.TimelineModel;
 import com.framgia.englishconversation.data.model.UserModel;
 import com.framgia.englishconversation.screen.createPost.CreatePostActivity;
+import com.framgia.englishconversation.screen.imageDetail.ImageDetailActivity;
```

Comment:
ERROR: Unused import: com.framgia.englishconversation.screen.imageDetail.ImageDetailActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

Reply:
Em dùng cho pull sau ạ,